pkg/operator/sync.go

```diff
@@ -39,24 +39,35 @@ type syncFunc struct {
 	fn   func(config *renderConfig) error
 }
 
+type syncErr struct {
+	task string
+	error
```

Comment:
let's call this `err error` - I'd avoid types embedding as much as we can

Reply:
people in Milan own style, don't they :D

Reply:
fantasy around naming above all lmao